### PR TITLE
Add s390x support

### PR DESCRIPTION
**Pull request by raulcabello**

### CODE REVIEW

**Comment by rohitsakala** [comment]
Add s390x support to rancher/rancher

Part of https://github.com/rancher/rancher/issues/35501

File: .drone.yml [hunk 1]
```diff
@@ -124,6 +124,9 @@ steps:
 
   - name: docker-publish
     image: rancher/drone-images:docker-s390x
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
```
Comment (jiaqiluo) [inline .drone.yml:129]: Why do we need the volume?  I do not see it is used in other docker-publish jobs for other arches.

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (thedadams) [reply .drone.yml:129]: The other `docker-publish` jobs use a `plugin` image. Since this job doesn't use a plugin image, then then volume is required.

**Review by thedadams — CHANGES_REQUESTED** [review]
The `master` branch of Rancher is not currently used and will be replaced. The target branch of this PR should be `release/v2.6`.

EDIT: Sorry, I misread this. Ignore my comment here.

**Review by thedadams — APPROVED** [review]

File: .drone.yml (hunk 1 above)
Comment (jiaqiluo) [reply .drone.yml:129]: ok, it makes sense. thanks for the explanation.

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Pull request merged**